.circleci/config.yml

```diff
@@ -12,6 +12,9 @@ parameters:
   git_release_tag:
     type: string
     default: ^v[0-9]+\.[0-9]+\.[0-9]+.*$
+  git_release_branch:
+    type: string
+    default: ^v[0-9]+\.[0-9]+$
```

Comment:
better to make it same as git_release_tag versioning `^v[0-9]+\.[0-9]+\.[0-9]+.*$`

Reply:
release branch doesn't contain the minor version and appendixes, we should strictly follow `vX.Y` imo.